tests/core/_orchestrator/test_orchestrator__submit.py

```diff
@@ -535,17 +534,14 @@ def test_submit_duration_development_mode():
     jobs = submission.jobs
     orchestrator.stop()
 
-    assert all(isinstance(job.execution_started_at, datetime) for job in jobs)
-    assert all(isinstance(job.execution_ended_at, datetime) for job in jobs)
+    assert all(isinstance(job.submitted_time, datetime) for job in jobs)
+    assert all(isinstance(job.run_time, datetime) for job in jobs)
+    assert all(isinstance(job.finished_time, datetime) for job in jobs)
     jobs_1s = jobs[0] if jobs[0].task.config_id == "task_config_id_1" else jobs[1]
     jobs_2s = jobs[0] if jobs[0].task.config_id == "task_config_id_2" else jobs[1]
     assert jobs_1s.execution_duration >= 1
     assert jobs_2s.execution_duration >= 2
 
-    assert submission.execution_duration >= 3
-    assert submission.execution_started_at == min(jobs_1s.execution_started_at, jobs_2s.execution_started_at)
-    assert submission.execution_ended_at == max(jobs_1s.execution_ended_at, jobs_2s.execution_ended_at)
-
 
 @pytest.mark.standalone
 def test_submit_duration_standalone_mode():
```

Comment:
This test is not only about submit duration anylonger. It should be renamed.

Reply:
Welp, it will be, when the attributes of a submissions entity is added.